### DISCUSSION TITLE
What do parentheses mean?

### DISCUSSION THREAD

**Discussion by philipdygeus** [CubeLabsNZ/CubeTime#159] [Q&A] [answered]
Thanks for a wonderful wonderful app. I'm only wondering why some results in the Best ao5 and Best Stats have parentheses and are greyed out? 
Thank you very much
Philip

**Comment by pdtxie** [comment, marked as answer]
In WCA competitions, averages of 5 are calculated based off removing the best and worst solves, and taking the average of the remaining three. The greyed solves are your best and worst, and are not included in the average

**Reply by philipdygeus** [reply]
Hi Tim and thank you very much for the explanation. I really appreciate it. best from the Netherlands
